src/main/java/dev/slimevr/bridge/WebSocketVRBridge.java

```diff
@@ -36,7 +36,7 @@
 	private final List<? extends ShareableTracker> shareTrackers;
 	private final List<ComputedTracker> internalTrackers;
 	
-	private final HMDTracker internalHMDTracker = new HMDTracker("itnernal://HMD");
+	private final HMDTracker internalHMDTracker = new HMDTracker("internal://HMD");
```

Comment:
I believe this may be a breaking change, would need to double check this

Reply:
It's not.